packages/vscode-extension/src/handlers.ts

```diff
@@ -681,8 +685,53 @@ export async function createNewEnvironment(args?: any[]): Promise<Result<Void, F
   return result;
 }
 
-export async function viewEnvironment(args?: any[]): Promise<Result<Void, FxError>> {
-  // todo add view logic
+export async function viewEnvironment(env: string): Promise<Result<Void, FxError>> {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
+    const envFilePath = environmentManager.getEnvConfigPath(
+      env,
+      workspace.workspaceFolders![0].uri.fsPath
+    );
+    const envPath: vscode.Uri = vscode.Uri.file(envFilePath);
+    if (await fs.pathExists(envFilePath)) {
+      vscode.workspace.openTextDocument(envPath).then(
+        (a: vscode.TextDocument) => {
+          vscode.window.showTextDocument(a, 1, false);
+        },
+        (error: any) => {
+          const openEnvError = new SystemError(
+            ExtensionErrors.OpenEnvProfileError,
+            `Can not open project environment ${env}.`,
```

Comment:
Better extract all the strings that might be visible to users to strings.json

Reply:
Fixed.